### PR TITLE
Salvador

### PR DESCRIPTION
**Pull request by Vesihiisi**
Mapping: https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/sv_(es)
Preview: https://www.wikidata.org/wiki/Wikidata:WikiProject_WLM/Mapping_tables/sv_(es)/preview

Task: https://phabricator.wikimedia.org/T176721

### CODE REVIEW

File: importer/SvEs.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    def update_descriptions(self):
+        eng = "cultural heritage site in El Salvador"
+        self.add_description("en", eng)
+
+    def set_heritage_id(self):
+        wlm = self.mapping["table_name"].upper()
+        self.add_statement("wlm_id", "{}-{}".format(wlm, self.id))
+
+    def exists_with_monument_article(self, language):
```
Comment (lokal-profil) [inline importer/SvEs.py]: Needs the #126 treatment

File: importer/SvEs.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            adm_q = utils.q_from_first_wikilink("es", municip_raw)
+        else:
+            municip_match = utils.get_item_from_dict_by_key(
+                dict_name=municip_dic,
+                search_in="itemLabel",
+                search_term=municip_raw)
+            if len(municip_match) == 1:
+                adm_q = municip_match[0]
+
+        if adm_q is None:
```
Comment (lokal-profil) [inline importer/SvEs.py]: stick an add to report here as well (since municipio failed)

Comment (lokal-profil) [reply importer/SvEs.py]: Done

File: importer/SvEs.py (hunk 1 above)
Comment (lokal-profil) [reply importer/SvEs.py]: Done

**Comment by Vesihiisi** [comment]
Could you update the preview? So that we can see some of the 'directions' at work.

**Comment by Vesihiisi** [comment]
Updated preview with local dump.

**Pull request merged**